python/lsst/analysis/drp/dataSelectors.py

```diff
@@ -28,46 +28,47 @@ def __call__(self, df, **kwargs):
         """Select on the given flags"""
```

Comment:
In this function, and all functions throughout, please document the input arguments and returned values. If you don't want to do this for all instances of `__call__` and that's the main method that does anything, you could probably put it in the class definition instead, I think that jives with the developer docs guide.

But in sum, if there's a list of arguments and/or a return statement, docstring time please!